test/input_json/TechnologyModels/pvwattsv8/dc_ac_ratio_0.02_PVWatts_Single_Owner_cmod_pvwattsv8.json

```diff
@@ -1,5 +1,5 @@
 {
-	"solar_resource_file" : "C:/Projects/Github/NREL/working/SAM/deploy/solar_resource/phoenix_az_33.450495_-111.983688_psmv3_60_tmy.csv",
+	"solar_resource_file" : "C:/Users/mprillim/sam_dev/SAM/deploy/solar_resource/phoenix_az_33.450495_-111.983688_psmv3_60_tmy.csv",
```

Comment:
It looks like this line was specific to Steve's computer before, and is now specific to yours, but flagging in case it should be set to a common file... but I don't think there's an easy way to use environment variables in JSON, is there @sjanzou ? I don't require this to be fixed before merging though.

Reply:
> It looks like this line was specific to Steve's computer before, and is now specific to yours, but flagging in case it should be set to a common file... but I don't think there's an easy way to use environment variables in JSON, is there @sjanzou ? I don't require this to be fixed before merging though.

@janinefreeman ,  thanks for flagging this - the JSON generated always has the specific test machine path... However, in the test body, the weather file is updated to use 
![image](https://github.com/user-attachments/assets/e2a9197f-3102-40c4-afdb-f7f65283639d)

and in the "Test" function, the weather file is set based on the SSCDIR/test/input_cases/general_data folder as
![image](https://github.com/user-attachments/assets/ef0b716e-4518-4536-811b-8d1a95962719)